cli/flags.rs

```diff
@@ -1022,7 +1022,7 @@ fn test_subcommand<'a, 'b>() -> App<'a, 'b> {
         .takes_value(true)
         .requires("unstable")
         .conflicts_with("inspect")
-        .conflicts_with("inspect-brk")
+        .conflicts_with("inspect-break")
```

Comment:
Not aliased? so breaking change.

Reply:
I am not too familiar with rust and would need help on how exactly to implement aliases. Else this could also wait till 2.x

Reply:
See https://github.com/denoland/deno/pull/7032/files#diff-abdd2d887e97d0eff6d352a1efd4dd953797699228a18a3a237d9818469da3cfR1326 for an example.

Reply:
added alias